.github/workflows/zero-downtime-tests.yml

```diff
@@ -2,7 +2,7 @@ name: ZeroDowntime Tests
 on:
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron: '0 1 * * 1' # run zero downtime tests at 1 AM (UTC), every monday (1)
+    - cron:  '0 4 * * 1-5' # run integration tests at 4 AM (UTC), monday to friday (1-5)
```

Comment:
was this change intentional?

Reply:
yes we discussed to have the test daily, but I'll rename it to zd